cluster/pulumi/tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "lib": [
       "es2021.string",
       "es2015.iterable",
-      "es2019.array"
+      "es2019.array",
+      "dom"
```

Comment:
why do we need this change? Requiring dom access in pulumi code doesn't make a lot of sense